### ISSUE TITLE
Updating buildSrcVersions itself

### ISSUE THREAD

**Issue by jmfayard** [jmfayard/buildSrcVersions#47]
- **Since**: 0.4.1
- **Only for build.gradle.kts**

--

Provided that you have run `./gradle buildSrcVersions` once with a version 0.4.1 or better, you can replace the plugin declaration with a typesafe accessor

```diff
plugins {
-  id("de.fayard.buildSrcVersions") version "0.4.1"
+ buildSrcVersions
}
```

Now you can update the plugin itself like any other dependency by updating `Versions.de_fayard_buildsrcversions_gradle_plugin`

**Comment by WhosNickDoglio** [comment]
Can this be generated for all the plugins in a project? 

I add [detekt](https://github.com/arturbosch/detekt) and Gradle Build scans to all my projects so it would be nice to automate their versioning as well. 👍

**Comment by jmfayard** [comment]
Perhaps but then only for Gradle version >= 5.6
See issue #60

Le jeu. 5 sept. 2019 à 00:14, Nicholas Doglio <notifications@github.com> a
écrit :

**Comment by jmfayard** [comment]
In version 0.6.0 there is a better solution provided that you use Gradle 5.6

See "gradle :refreshVersions" generates gradle.properties with versions and available updates #77
https://github.com/jmfayard/buildSrcVersions/issues/77
